# 第1章 物质的基本性质

In [1]:
import sympy as sp

sp.init_session()

IPython console for SymPy 1.14.0 (Python 3.13.9-64-bit) (ground types: python)

These commands were executed:
>>> from sympy import *
>>> x, y, z, t = symbols('x y z t')
>>> k, m, n = symbols('k m n', integer=True)
>>> f, g, h = symbols('f g h', cls=Function)
>>> init_printing()

Documentation can be found at https://docs.sympy.org/1.14.0/



## 物质的基本结构及形态


## 实物的基本性质

## 场与物质的相互作用

## 物质的能量

## 思考题

1.1 有人认为，实物是实的，场是虚的，在我们的现实生活中起主要作用的物质是实物，场的存在与否微不足道。你认为这种观点对吗？为什么？

1.2 生命物质系统中是否有特定场存在，其特征是什么？试举例说明。

1.3 对于保守场，人们引入势的概念来描述其特征，这是否意味着在其他情况下，就不能引入势的概念？为什么？举例说明。

1.4 质量与能量等效原理，可由爱因斯坦的质能公式 $\Delta E=\Delta mc^2$ 表述，也就是说，若物质获得 $\Delta E$ 的能量，则其质量就增加了 $\Delta m$，应如何正确理解？

1.5 有人提出，物理学常用于研究简单的系统，而生物学则研究自然界最复杂的生命，因此，两者之间没有任何必然的、内在的联系，你的看法如何？